sklearn/utils/testing.py

```diff
@@ -597,15 +626,15 @@ def all_estimators(include_meta_estimators=False,
         not be default-constructed sensibly. These are currently
         Pipeline, FeatureUnion and GridSearchCV
 
-    include_dont_test : boolean, default=False
-        Whether to include "special" label estimator or test processors.
-
     type_filter : string, list of string,  or None, default=None
         Which kind of estimators should be returned. If None, no filter is
         applied and all estimators are returned.  Possible values are
         'classifier', 'regressor', 'cluster' and 'transformer' to get
         estimators only of these specific types, or a list of these to
         get the estimators that fit at least one of the types.
+        Whether to include "special" label estimator or test processors.
+
+    include_dont_test : boolean, default=False
```

Comment:
It looks like there is a small error in the change that you made to the docstring above.

Reply:
Good catch, thanks